<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
	background-image: url(:/images/login-window.png);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_sign_up">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:#4062b2;
	border-radius:6px;
	color:#ffffff;
	font-family:Arial;
	font-size:11px;
	font-weight:bold;
	padding:5px 5px;
}

QPushButton:hover {
	background-color:#96abda;
}
</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_username_login">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Fjalla One', sans-serif;
font-size: 12px;
color: #ffffff;
font-weight: bold;
text-decoration: none;
font-style: normal;
</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Fjalla One', sans-serif;
font-size: 12px;
color: #4C6FBF;
font-weight: bold;
text-decoration: none;
font-style: normal;
</string>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_name">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color:#cfd8ee;
	border-radius:4px;
	border:1px solid #4c6fbf;
	cursor:text;
	color:#324e8c;
	font-family:Impactl;
	font-size:11px;
	font-weight:normal;
	padding:2px 5px;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_login">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:#bcc9e7;
	border-radius:6px;
	color:#39589f;
	font-family:Arial;
	font-size:11px;
	font-weight:bold;
	padding:5px 5px;
}

QPushButton:hover {
	background-color:#96abda;
}
</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Fjalla One', sans-serif;
font-size: 12px;
color: #ffffff;
font-weight: bold;
text-decoration: none;
font-style: normal;
</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Fjalla One', sans-serif;
font-size: 16px;
color: #4C6FBF;
font-weight: bold;

</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Fjalla One', sans-serif;
font-size: 12px;
color: #4C6FBF;
font-weight: bold;
text-decoration: none;
font-style: normal;
</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Fjalla One', sans-serif;
font-size: 18px;
color: #ffffff;
font-weight: bold;
text-decoration: none;
font-style: normal;
</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_pass_login">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Fjalla One', sans-serif;
font-size: 12px;
color: #4C6FBF;
font-weight: bold;
text-decoration: none;
font-style: normal;
</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Fjalla One', sans-serif;
font-size: 12px;
color: #4C6FBF;
font-weight: bold;
text-decoration: none;
font-style: normal;
</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_pass">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color:#cfd8ee;
	border-radius:4px;
	border:1px solid #4c6fbf;
	cursor:text;
	color:#324e8c;
	font-family:Impactl;
	font-size:11px;
	font-weight:normal;
	padding:2px 5px;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_confirm">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color:#cfd8ee;
	border-radius:4px;
	border:1px solid #4c6fbf;
	cursor:text;
	color:#324e8c;
	font-family:Impactl;
	font-size:11px;
	font-weight:normal;
	padding:2px 5px;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_username">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color:#cfd8ee;
	border-radius:4px;
	border:1px solid #4c6fbf;
	cursor:text;
	color:#324e8c;
	font-family:Impactl;
	font-size:11px;
	font-weight:normal;
	padding:2px 5px;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Fjalla One', sans-serif;
font-size: 12px;
color: red;
font-weight: bold;
text-decoration: none;
font-style: normal;
</string>
    </property>
    <property name="text">
     <string>Error</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
